ts/src/ascendex.ts

```diff
@@ -3087,6 +3087,85 @@ export default class ascendex extends Exchange {
         return 'failed';
     }
 
+    async fetchFundingHistory (symbol: string = undefined, since: Int = undefined, limit: Int = undefined, params = {}) {
```

Comment:
@Dan-krm Great addition, I'm not so sure about the name since it's very similar to `fetchFundingRateHistory` but I think we can keep it for now and change it in the future if someone comes up with a better alternative.